sdk/contentsafety/azure-ai-contentsafety/tests/test_content_safety.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
We need to have async tests and samples. Let's not block this PR on that but please let's get it in in a separate PR.